getting-started/kubearmor_visibility.md

```diff
@@ -0,0 +1,128 @@
+# KubeArmor Visibility
+
+KubeArmor currently supports enabling visibility for containers/hosts
+Visibility for containers is enabled by default and can be accesed using `karmor` tool
+
+
+
+## Getting Container Visibility
+### Example : multiubuntu
```

Comment:
I would rather suggest using wordpress-mysql as an example since it is more realistic.